docs/design/features/hybrid-globalization.md

```diff
@@ -18,3 +18,157 @@ Affected public APIs:
+let changedLocale = "ja-JP"
+let changedStr1 = hiraganaBig.toLocaleLowerCase(changedLocale);
+let changedStr2 = katakanaSmall.toLocaleLowerCase(changedLocale);
+changedStr1.localeCompare(changedStr2, changedLocale)                        // 0;
+```
+
+From this reason, comparison with locale `ja-JP` `CompareOption` `IgnoreCase` and `StringComparison`: `CurrentCultureIgnoreCase` and `InvariantCultureIgnoreCase` behave like a combination `IgnoreCase | IgnoreKanaType` (see: previous point about `IgnoreKanaType`). For other locales the behavior is unchanged with the following known exceptions:
+
+|                  **character 1**                 |                       **character 2**                      | **CompareOptions**                | **hybrid globalization** | **icu** |
+|:------------------------------------------------:|:----------------------------------------------------------:|-----------------------------------|:------------------------:|:-------:|
+| `\uFF9E`  (HALFWIDTH KATAKANA VOICED SOUND MARK) | `\u3099`   (COMBINING KATAKANA-HIRAGANA VOICED SOUND MARK) | None / IgnoreCase / IgnoreSymbols |             1            |    0    |
+
+- `IgnoreNonSpace`
+
+`IgnoreNonSpace` cannot be used separately without `IgnoreKanaType`. Argument `sensitivity: "case"` is used for comparison and it ignores both types of characters. Option `IgnoreNonSpace` alone throws `PlatformNotSupportedException`.
+
+```
+let hiraganaAccent = `${String.fromCharCode(12353)} á`                           // %u3041 = ぁ
+let katakanaNoAccent = `${String.fromCharCode(12449)} a`                         // %u30A1 = ァ
+hiraganaAccent.localeCompare(katakanaNoAccent, "en-US", { sensitivity: "case" }) // 0; case:  a ≠ b, a = á, a ≠ A
+```
+
+- `IgnoreNonSpace | IgnoreCase`
+Combination of `IgnoreNonSpace` and `IgnoreCase` cannot be used without `IgnoreKanaType`. Argument `sensitivity: "base"` is used for comparison and it ignores three types of characters. Combination `IgnoreNonSpace | IgnoreCase` alone throws `PlatformNotSupportedException`.
+
+```
+let hiraganaBigAccent = `${String.fromCharCode(12353)} A á`                              // %u3041 = ぁ
+let katakanaSmallNoAccent = `${String.fromCharCode(12449)} a a`                          // %u30A1 = ァ
+hiraganaBigAccent.localeCompare(katakanaSmallNoAccent, "en-US", { sensitivity: "base" }) // 0; base: a ≠ b, a = á, a = A
+```
+
+- `IgnoreSymbols`
+
+The subset of ignored symbols is limited to the symbols ignored by `string1.localeCompare(string2, locale, { ignorePunctuation: true })`. E.g. currency symbols, & are not ignored
+
+```
+let hiraganaAccent = `${String.fromCharCode(12353)} á`                     // %u3041 = ぁ
+let katakanaNoAccent = `${String.fromCharCode(12449)} a`                   // %u30A1 = ァ
+hiraganaBig.localeCompare(katakanaSmall, "en-US", { sensitivity: "base" }) // 0; base: a ≠ b, a = á, a = A
+```
+
+- List of all `CompareOptions` combinations always throwing `PlatformNotSupportedException`:
```

Comment:
In the future, could we use attributes or a compile time analyzer to make builds fail when the user's code attempts to use these combinations? (Obviously not in this PR)